packages/spec-parser/src/specParser.ts

```diff
@@ -328,28 +328,31 @@ export class SpecParser {
       }
       await fs.outputFile(outputSpecPath, resultStr);
 
-      for (const url in newSpec.paths) {
-        for (const method in newSpec.paths[url]) {
-          // paths object may contain description/summary, so we need to check if it is a operation object
-          if (method === ConstantString.PostMethod || method === ConstantString.GetMethod) {
-            const operation = (newSpec.paths[url] as any)[method] as OpenAPIV3.OperationObject;
-            try {
-              const [card, jsonPath] = AdaptiveCardGenerator.generateAdaptiveCard(operation);
-              const fileName = path.join(adaptiveCardFolder, `${operation.operationId!}.json`);
-              const wrappedCard = wrapAdaptiveCard(card, jsonPath);
-              await fs.outputJSON(fileName, wrappedCard, { spaces: 2 });
-              const dataFileName = path.join(
-                adaptiveCardFolder,
-                `${operation.operationId!}.data.json`
-              );
-              await fs.outputJSON(dataFileName, {}, { spaces: 2 });
-            } catch (err) {
-              result.allSuccess = false;
-              result.warnings.push({
-                type: WarningType.GenerateCardFailed,
-                content: (err as Error).toString(),
-                data: operation.operationId!,
-              });
+      if (isMe === undefined || isMe === true) {
```

Comment:
How about to make adaptiveCardFolder optional?  If adaptiveCardFolder is undefined, then it will not generate adaptive cards.

Another suggestion is that currently we plan to support 3 different project types: copilot, teams ai, and SME, so it is better to use an Enum for project types?

Reply:
I think you can just merge it, and I will optimize the whole spec parser lib